api/client/client.go

```diff
@@ -3128,6 +3128,33 @@ func (c *Client) DeleteAutoUpdateAgentRollout(ctx context.Context) error {
 	return trace.Wrap(err)
 }
 
+func (c *Client) TriggerAutoUpdateAgentGroup(ctx context.Context, groups []string, state autoupdatev1pb.AutoUpdateAgentGroupState) (*autoupdatev1pb.AutoUpdateAgentRollout, error) {
```

Comment:
Why do we need dedicated methods here rather than adding a method that returns an `AutoUpdateServiceClient`?